x-pack/osquerybeat/Jenkinsfile.yml

```diff
@@ -76,3 +76,10 @@ stages:
         e2e:
             enabled: false
         stage: packaging
+    packaging-arm:
+        packaging-arm: "mage package"
+        e2e:
+            enabled: false
+        platforms:             ## override default label in this specific stage.
+          - "arm"
+        stage: packaging
```

Comment:
Enable the arm package stage for the `x-pack/osquerybeat`. This stage will run as part of the default pipeline.